cmd/openshift-install/create.go

```diff
@@ -352,6 +364,7 @@ func waitForInitializedCluster(ctx context.Context, config *rest.Config) error {
 	defer cancel()
 
 	failing := configv1.ClusterStatusConditionType("Failing")
+	timer.StartTimer("Cluster Operator")
```

Comment:
`Cluster Operator` -> `Cluster Operators`

Reply:
Fixed it.